renderchan/utils.py

```diff
@@ -186,3 +186,12 @@ def ini_wrapper(filename):
 def is_true_string(string):
     string = string.lower()
     return string == "1" or string == "true" or string == "yes"
+
+
+def sanitize_path(path):
+    assert isinstance(path, str)
+    if os.name == 'nt':
+        path = path.replace('/', os.sep)
+    else:
+        path = path.replace('\\', os.sep)
```

Comment:
Now this will break posix paths with `\`, and possibly windows paths with `/`. This doesn't seem like a very robust solution.

Reply:
I just cannot find better solution...
Some versions of Synfig returns paths with Windows slashes, some return Linux slashes... That depends on which system the file was edited.

Reply:
I think it should be handled by the synfig module in exactly the same way that Synfig handles ambiguous input.